Formula/gitlint.rb

```diff
@@ -3,8 +3,8 @@ class Gitlint < Formula
 
   desc "Linting for your git commit messages"
   homepage "https://jorisroovers.github.io/gitlint"
-  url "https://files.pythonhosted.org/packages/9a/0c/bacbf0ea52b924ff7d6984b2756e544d0e276c56663bb37e0c08781d4ad3/gitlint-0.16.0.tar.gz"
-  sha256 "30ee2bdae611bbf66df6326b5da1afc14bf0be337e1d3021fafeb7f13b37f55b"
+  url "https://files.pythonhosted.org/packages/92/15/8850b30ac84e47ae7af0d50b63da171c19a63fa947139efa8de7a3cbc9a6/gitlint-0.17.0.tar.gz"
+  sha256 "8c10c6b404d255b43ddc4a2f5f13bcb10284bc162adfb2c03b10708309009189"
```

Comment:
> The solution would be to install `gitlint-core` under the name `gitlint`

Can we just replace this with the `gitlint-core` sdist tarball?

Reply:
Thanks for the suggestion @carlocab. Something went a bit wrong with the force push, so I'll sort that out later.

Do you have any thoughts on dealing with `HEAD`, since it should clone a subdirectory of the project: https://github.com/jorisroovers/gitlint/tree/main/gitlint-core. Thank you for your help.

Reply:
`cd "gitlint-core" if build.head?` might work.